std_msgs/Header header
stretch_mover/KnownObject[] objects